maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FinancialAssessmentDetailEntity.java

```diff
@@ -24,28 +27,63 @@ public class FinancialAssessmentDetailEntity {
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "fin_ass_det_seq")
     @Column(name = "ID")
     private Integer id;
-    @Column(name = "FINA_ID")
-    private Integer financialAssessmentId;
+
     @Column(name = "CRITERIA_DETAIL_ID")
     private Integer criteriaDetailId;
+
     @CreationTimestamp
     @Column(name = "DATE_CREATED", nullable = false, updatable = false)
     private LocalDateTime dateCreated;
+
     @Column(name = "USER_CREATED", nullable = false, updatable = false)
     private String userCreated;
+
     @Builder.Default
     @Column(name = "APPLICANT_AMOUNT")
     private BigDecimal applicantAmount = BigDecimal.valueOf(0);
+
     @Column(name = "APPLICANT_FREQ")
     private Frequency applicantFrequency;
+
     @Builder.Default
     @Column(name = "PARTNER_AMOUNT")
     private BigDecimal partnerAmount = BigDecimal.valueOf(0);
+
     @Column(name = "PARTNER_FREQ")
     private Frequency partnerFrequency;
+
     @UpdateTimestamp
     @Column(name = "DATE_MODIFIED")
     private LocalDateTime dateModified;
+
     @Column(name = "USER_MODIFIED")
     private String userModified;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "FINA_ID", nullable = false)
+    private FinancialAssessmentEntity financialAssessment;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Just in terms of coding standards we should avoid single line `if`s.

Reply:
This part was auto-generated by the IDE 😆, but I agree, it's not great practice.